jazz-ui/src/app/pages/service-cost/service-cost.component.ts

```diff
@@ -183,33 +186,30 @@ export class ServiceCostComponent implements OnInit {
 		
 	}
 
-	accList=['tmodevops','tmonpe'];
-  regList=['us-west-2', 'us-east-1'];
-	accSelected:string = 'tmodevops';
-	regSelected:string = 'us-west-2';
+	accList=env_internal.urls.accounts;
+	regList=env_internal.urls.accounts;
```

Comment:
`regList` is region, so should be referencing a diff key than `env_internal.urls.accounts`

Reply:
I have fixed this issue, Updated the PR